motoko/hello-angular-motoko/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "hello-angular-motoko",
+  "version": "0.0.3",
+  "author": {
+    "name": "Shawn Dotey",
+    "email": "insight431@protonmail.com",
+    "url": "https://forum.dfinity.org/u/infinityio.tech"
+  },
+  "description": "An example of a Motoko project with an Angular 14 frontend",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "TypeScript",
+    "Angular",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "",
+    "prebuild": "",
+    "prestart": "",
+    "init:dfx": "npm run restart:clean:dfx && npm run refresh:motoko && dfx stop",
+    "start": "npm run restart:dfx && npm run refresh:motoko && npm run serve:both && npm run print:ng",
+    "test": "npm run restart:dfx && ng test",
+    "deploy:dfx": "npm run restart:dfx && npm run build:prod && dfx deploy --network ic --no-wallet",
+    "serve:both": "concurrently -k \"npm run watch:motoko\" \"ng serve && dfx stop\"",
+    "watch:motoko": "nodemon --watch src --ext mo -x \"npm run refresh:motoko\"",
+    "print:dfx:www": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id www)",
+    "print:dfx:motoko": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id motoko)",
+    "generate:motoko": "dfx generate motoko",
+    "restart:dfx": "dfx stop && dfx start --background",
+    "restart:clean:dfx": "dfx stop && dfx start --background --clean",
+    "refresh:motoko": "dfx deploy motoko && npm run copy:dfx",
```

Comment:
why is `copy:dfx` included here? And why _after_ the deploy? I don't understand what the copy step does here.

Reply:
`refresh:motoko` is used when initializing and running the project in development. The `copy:dfx` is the same as the `copy:types` command generated by a default new project (`dfx new <project_name> --frontend), except it removes typescript files as they currently have an issue with trailing commas. This command is used to refresh the front end canister declarations after a hot reload and on the project initialization. Otherwise, they are not available for the the front end to consume.